src/services/users.service.ts

```diff
@@ -260,6 +261,27 @@ export class UsersService {
 
     return this.axios.get(`${this.config.DRIVE_NEW_GATEWAY_URL}/gateway/users`, requestConfig);
   }
+
+  async addUserTierToVPNDatabase(userUuid: User['uuid'], tier: VpnFeatures['featureId']): Promise<void> {
```

Comment:
`activateVPNTier` or `enableVPNTier` would be a better naming. You do not - and you should not - know how the VPN service is handling this tier. Specifying - and assuming - how an external service manages something on the naming of a function that is not related, is tech debt. In fact, this function is not adding any user to the VPN database, it's sending a HTTP request to the VPN service to enable the user's tier on that service